Examples/SearchExamples.xcodeproj/project.pbxproj

```diff
@@ -439,33 +437,38 @@
 /* End XCConfigurationList section */
 
 /* Begin XCRemoteSwiftPackageReference section */
-		FE0720BC26FC86C70065A273 /* XCRemoteSwiftPackageReference "search-ios" */ = {
+		04F0A15F2B4F276D0097D316 /* XCRemoteSwiftPackageReference "mapbox-maps-ios" */ = {
 			isa = XCRemoteSwiftPackageReference;
-			repositoryURL = "https://github.com/mapbox/search-ios";
+			repositoryURL = "https://github.com/mapbox/mapbox-maps-ios.git";
 			requirement = {
 				kind = upToNextMajorVersion;
-				minimumVersion = "1.0.0-beta";
+				minimumVersion = 10.0.0;
 			};
 		};
-		FE93032326F8D87A00644DF6 /* XCRemoteSwiftPackageReference "mapbox-maps-ios" */ = {
+		FE0720BC26FC86C70065A273 /* XCRemoteSwiftPackageReference "search-ios" */ = {
 			isa = XCRemoteSwiftPackageReference;
-			repositoryURL = "git@github.com:mapbox/mapbox-maps-ios.git";
+			repositoryURL = "https://github.com/mapbox/search-ios";
 			requirement = {
 				kind = upToNextMajorVersion;
-				minimumVersion = "10.0.0-rc";
+				minimumVersion = "1.0.0-rc.8";
```

Comment:
Maybe this should be a local SPM package, or Examples app won't represent the actual state of Search SDK on each commit.

Reply:
Yeah that's a great idea. I'll split that change into a follow-up ticket / PR to keep this one moving along.

Reply:
→ SSDK-525